tests/integration/cmor/_fixes/native6/test_era5.py

```diff
@@ -89,14 +114,28 @@ def _era5_time(frequency):
         timestamps = [788928, 788929, 788930]
     elif frequency == 'monthly':
         timestamps = [788928, 789672, 790344]
-    return iris.coords.DimCoord(np.array(timestamps, dtype='int32'),
-                                standard_name='time',
-                                long_name='time',
-                                var_name='time',
-                                units=Unit(
-                                    'hours since 1900-01-01'
-                                    '00:00:00.0',
-                                    calendar='gregorian'))
+    return iris.coords.DimCoord(
+        np.array(timestamps, dtype='int32'),
+        standard_name='time',
+        long_name='time',
+        var_name='time',
+        units=Unit('hours since 1900-01-01'
+                   '00:00:00.0', calendar='gregorian'),
+    )
+
+
+def _era5_plev():
+    values = np.array([
+        1,
+        1000,
+    ])
+    return iris.coords.DimCoord(
+        values,
+        long_name="pressure",
+        units=Unit("millibars"),
```

Comment:
The cf_units don't recognize `hPa`?

Reply:
It supports both, and can convert between them.

Reply:
This is what ERA5 data looks like, e.g. the output from `ncdump` on `ta`:
```
netcdf era5_temperature_1990_monthly {
dimensions:
	longitude = 1440 ;
	latitude = 721 ;
	level = 37 ;
	time = 12 ;
variables:
	float longitude(longitude) ;
		longitude:units = "degrees_east" ;
		longitude:long_name = "longitude" ;
	float latitude(latitude) ;
		latitude:units = "degrees_north" ;
		latitude:long_name = "latitude" ;
	int level(level) ;
		level:units = "millibars" ;
		level:long_name = "pressure_level" ;
	int time(time) ;
		time:units = "hours since 1900-01-01 00:00:00.0" ;
		time:long_name = "time" ;
		time:calendar = "gregorian" ;
	short t(time, level, latitude, longitude) ;
		t:scale_factor = 0.00206523785420788 ;
		t:add_offset = 248.702573032928 ;
		t:_FillValue = -32767s ;
		t:missing_value = -32767s ;
		t:units = "K" ;
		t:long_name = "Temperature" ;
		t:standard_name = "air_temperature" ;
```

Reply:
Ah okay. Since I say hPa on the ERA5 documentation page